src/test/java/com/bartek/Medical/Model/controller/PatientControllerTest.java

```diff
@@ -70,13 +86,22 @@ void addPatient_PatientExist_ReturnedPatient() throws Exception {
                 .andExpect(content().contentType(MediaType.APPLICATION_JSON))
                 .andExpect(MockMvcResultMatchers.jsonPath("$.email").value("john.doe@example.com"));
     }
+
     @Rollback
     @Test
     void deletePatientbyEmail_PatientExist_PatientDeleted() throws Exception {
```

Comment:
Ten test ma przetestowac usuwanie pacjenta. Normalnym wiec jest ze taki test powinien testowac tylko usuwanie. Zastanowmy sie teraz co sie stanie jesli nie zadziala nam GET -> sfailuje nam test o deletea przez to ze nie dziala get. Naturalnym bedzie ze ktos uzna ze nie dziala metoda delete. I jak wejdzie w taki kod to pomysl wt co chlop robi lol